applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -104,7 +104,7 @@ const useStyles = makeStyles((theme) => ({
       paddingRight: 0,
       marginTop: theme.spacing(2),
       [theme.breakpoints.up("sm")]: {
-        maxHeight: "calc(100vh - 19.5rem)",
+        maxHeight: "calc(100vh - 15.0rem)",
```

Comment:
this is not the better way to fit the vertical space. Better use flex with stretch